spec/features/reviews_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "recenzje" do
+
```

Comment:
Brak logowania do aplikacji. Ponadto proszę zrobić to na bazie istniejących testów dla recenzji, które już są w gałęzi `master`.